src/flows/pipes/RawFluxEditor/FunctionToolTipContent.tsx

```diff
@@ -0,0 +1,60 @@
+// Libraries
```

Comment:
was this file just renamed or were there changes made to it?

Reply:
the previous file had two components:
1.  component `FunctionTooltipContents` was unchanged, and placed inside a newly [explicitly] named file.
2. component `ToolbarFunction` was a reusable element, used for popover/injection element for each list item. this became a new shared component `FluxInjectionOption`.

The choice of keeping these two separate is so that the `FunctionTooltipContents` can be modified/extract into a new shared FilteredList component for src/flows:
* Then both inject-formula and inject-secrets will use this new FilterList.
* But the FilterList will take any child component (not just a FluxOption to be injected). So that we can reuse it again in src/flows.

That was the plan...and I can do that now (in this PR).

Reply:
Turns out that we had 3 lists (used in the RawFluxEditor sidebar):
1. SecretsList
2. functions
3. dynamicFunctions

`functions` was a filtered "list" which was not an array. Rather, it is grouped functions (by category) with each category have it's own list -- which is filtered. Therefore the logic for both filtering & rendering is different for the `functions`.  Plus it's name is confusing.

Here is the updated components:
1. FilterList in `src/flows/shared`. Handles search filtered arrays.
2. SecretList in `src/flows/pipes/RawFluxEditor`. uses FilterList.
3. functions is now GroupedFunctionsList, in `src/flows/pipes/RawFluxEditor`. filtering is object based, not array. So it's own code.
4. dynamicFunctions is now FunctionsList, in `src/flows/pipes/RawFluxEditor`. uses FilterList.